package.json

```diff
@@ -39,6 +39,7 @@
         "moment": "2.24.0",
         "ng-jhipster": "0.9.3",
         "ng-pick-datetime": "7.0.0",
+        "ng-pick-datetime-moment": "^1.0.8",
```

Comment:
Please do not use `^` as it can lead to build issues when the dependency is updated

Reply:
Thanks, you are right!